packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -94,7 +94,7 @@ const chatMessageStyles: ComponentSlotStylesPrepared<
     borderRadius: v.borderRadius,
     boxShadow: v.actionMenuBoxShadow,
     // we need higher zIndex for the action menu in order to be displayed above the focus border of the chat message
-    zIndex: 1000,
+    zIndex: v.zIndexOverlay,
```

Comment:
@mnajdova should it be `overlayZIndex`?

```suggestion
    zIndex: v.overlayZIndex,
```

Reply:
Sure, will update it ;)